scripts/update_to_1.25.0.php

```diff
@@ -19,6 +20,11 @@ function apply_update() {
 		FRESHRSS_PATH . '/extensions',
 	);
 
+	// check PHP version requirement
+	if (version_compare(MIN_PHP_VERSION, phpversion(), '>')) {
```

Comment:
Do you prefer `phpversion()` or `PHP_VERSION`?

Reply:
`PHP_VERSION` as this is what we use elsewhere, but I do not mind